OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -2753,7 +2779,9 @@ public void testSessionTimeTrackingOnPrivacyConsent() throws Exception {
 
       // Provide consent to OneSignal SDK amd forward time 2 minutes
       OneSignal.provideUserConsent(true);
-      threadAndTaskWait();
+      // TODO: This passed without a resume here, why?
+      blankActivityController.resume();
```

Comment:
This test shouldn't have needed to be modified to pass.

Reply:
Thats what I thought too, gonna dig in a little more